datastore/elastic_search/query.py

```diff
@@ -282,17 +283,27 @@ def full_text_query(connection, index_name, doc_type, entity_name, sentence, fuz
          u'mumbai': u'mumbai',
          u'pune': u'pune'}
     """
-    data = _generate_es_search_dictionary(entity_name, sentence, fuzziness_threshold,
-                                          language_script=search_language_script)
-    kwargs = dict(kwargs, body=data, doc_type=doc_type, size=constants.ELASTICSEARCH_SEARCH_SIZE, index=index_name)
+    data = [
+        _generate_es_search_dictionary(entity_name, sentence_, fuzziness_threshold,
+                                       language_script=search_language_script) for sentence_ in sentence]
+    # kwargs = dict(kwargs, body=data, doc_type=doc_type, size=constants.ELASTICSEARCH_SEARCH_SIZE, index=index_name)
+    query_data = []
+    index_for_each_query = {}
```

Comment:
This should be equal to
`{'index': index_name, 'type': doc_type}'`

Reply:
Done